src/components/ModalTableConfirm/ModalTableConfirm.js

```diff
@@ -0,0 +1,13 @@
+import './ModalTableConfirm.css';
+
+const ModalTableConfirm = ({ modalShow, deleteItem, closeModal }) => {
```

Comment:
Its better to incapsulate all modal logic in this component. You can create state here and function to toggle this state near.